redirects/17.0.txt

```diff
@@ -96,6 +96,7 @@ applications/marketing/sms_marketing/pricing/pricing_and_faq.rst applications/ma
 applications/sales/crm/acquire_leads/generate_leads.rst applications/sales/crm/acquire_leads/email_manual.rst                                             #generate_leads --> email_manual
 applications/sales/crm/pipeline/multi_sales_team.rst applications/sales/crm/pipeline/manage_sales_teams.rst                                               # multi_sales_team --> manage_sales_teams
 applications/sales/point_of_sale/pricing/digital_labels.rst applications/sales/point_of_sale/pricing/electronic_labels.rst                                # digital_labels --> electronic_labels
+applications/sales/point_of_sale/payment_methods/terminals/vantiv.rst applications/sales/point_of_sale/payment_methods/terminals.rst		#remove vantiv integration
```

Comment:
Nitpick for the redirect comment :) I suggest we stick with the existing convention of describing the redirect using arrows
```suggestion
applications/sales/point_of_sale/payment_methods/terminals/vantiv.rst applications/sales/point_of_sale/payment_methods/terminals.rst		# terminals/vantiv --> terminals
```